### PR TITLE
Ruby support for the buildpack

### PR DESCRIPTION
**Pull request by hmadison**
Adds Ruby support to the Google Cloud Platform buildpack.

### CODE REVIEW

File: ci/pipeline.yml [hunk 1]
```diff
@@ -361,12 +393,23 @@ jobs:
...
+      - put: instana-ruby-dependencies-version
+        params:
+          file: instana-ruby-gem/version
 
   - name: build-package-builder-image
     max_in_flight: 1
     plan:
       - get: cnb-package-builder-source
         trigger: true
+      - get: ruby-autotrace
```
Comment (cedricziel) [inline ci/pipeline.yml:428]: why do we get this here?

Comment (hmadison) [reply ci/pipeline.yml:428]: This is downloading the shim code we use to boot the gem without it being included in the Gemfile. Its bundled into the buildpack package.

File: google-cloud-platform/cloud-run.version [hunk 2]
```diff
@@ -1 +1 @@
-1.203.1-dev.1
+1.204.0
```
Comment (mdonkers) [inline google-cloud-platform/cloud-run.version]: I think the pipeline auto-bumps this version, so you shouldn't need to change it.
(Judging by the 'Stan' user updating the file)

Comment (hmadison) [reply google-cloud-platform/cloud-run.version]: For some reason, the pipeline is bumping branches too. I'll revert the Stan commits locally and push up a clean copy to merge.

Comment (mdonkers) [reply google-cloud-platform/cloud-run.version]: Overlooked this was changed by Stan. That robot is sometimes too smart :-)

**Comment by hmadison** [comment]
@mdonkers I've pushed a "clean" version of this branch.

**Pull request closed without merging**